package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "scripts": {
     "clean": "rimraf dist distTest libEs5 libEs6",
-    "lint": "eslint --fix \"*.js\" \"src/**/*.js\" \"test/**/*.js\"",
+    "lint": "eslint --cache --fix \"*.js\" \"{src,test}/**/*.js\"",
```

Comment:
Enabling caching means eslint will only run on files that have changed since the last time eslint was run. This speeds things up.

Also, one of the reasons I upgraded eslint is because it now supports brace expansion (`{src,test}`).